mmengine/runner/runner.py

```diff
@@ -687,6 +687,43 @@ def set_randomness(self,
             deterministic=deterministic,
             diff_rank_seed=diff_rank_seed)
 
+    def set_torch_cfg(self, torch_cfg: dict) -> None:
+        """Set torch variable to by config.
+
+        Args:
+            torch_cfg (dict): Contains key-value pair which defines the names
+                of torch variables and corresponding values.
+
+        Examples:
+            >>> torch_cfg = dict(
+            >>>    backends.cuda.matmul.allow_tf32=True,
+            >>>    torch.backends.cuda.matmul.allow_fp16_reduced_precision_reduction=True
+            >>>    )
+        """  # noqa: E501
+        for attributes, value in torch_cfg.items():
+            assert isinstance(attributes, str)
+            if attributes in [
+                    'backends.cudnn.benchmark', 'backends.cudnn.deterministic',
+                    'torch.backends.cudnn.benchmark',
+                    'torch.backends.cudnn.deterministic'
+            ]:
+                raise ValueError(
+                    'backends.cudnn.benchmark or backends.cudnn.deterministic '
+                    'should be set in `randomness_cfg`, rather than '
+                    '`torch_cfg`')
+            attributes_list = attributes.split('.')
+            modules = attributes_list[:-1]
+            attribute = attributes_list[-1]
+            target_module = torch
+            try:
+                for module in modules:
+                    target_module = getattr(target_module, module)
```

Comment:
the example docstring shows both `torch.xxx` and `xxx.xx` are supported. But this may cause `torch.torch` error.